docs/language/operators/from.md

```diff
@@ -169,3 +174,22 @@ zed -lake example query -z '
 {flip:1,result:"heads",word:"one"}
 {flip:2,result:"tails",word:"two"}
 ```
+
+_Use `pass` to combine our join output with data from yet another source_
+```mdtest-command
+zed -lake example query -z '
+  from (
+    pool coinflips => sort flip
+    pool numbers => sort number
+  ) | join on flip=number word
+  | from (
+    pass
+    pool coinflips@trial => c:=count() | c:=int64(c) | yield "There were ${c} trial flips"
```

Comment:
I had to do `c:=count() | c:=int64(c)` instead of `c:=int64(count())` because of #4539.